src/platform/qt/Window.cpp

```diff
@@ -1054,6 +1073,17 @@ void Window::setupMenu(QMenuBar* menubar) {
 	}
 	m_config->updateOption("frameskip");
 
+	QMenu* rotateMenu = avMenu->addMenu("Rotate screen");
+	ConfigOption* rotateOption = m_config->addOption("rotation");
+	rotateOption->connect([this](const QVariant& value) {
+		rotateFrame(value.toInt() * 90);
```

Comment:
I'd prefer the actual values be used in the options, as opposed to multiplying up by 90, so that finer values can be used if desired in the future.